File: crates/deslop/tests/fixtures/fused-golden-php/rename_b.php
<?php

function dispatch(int $mass, int $span, string $handler): string
{
    $rating = $mass * 3 + $span;
    if ($rating > 900) {
        return $handler . '-freight';
    }

    if ($rating > 400) {
        return $handler . '-ground';
    }

    return $handler . '-parcel';
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/rename_b.cs
namespace Golden.Rename;

public static class ParcelDispatcher
{
    public static string Dispatch(int mass, int span, string handler)
    {
        int rating = mass * 3 + span;
        if (rating > 900)
        {
            return handler + "-freight";
        }

        if (rating > 400)
        {
            return handler + "-ground";
        }

        return handler + "-parcel";
    }
}
